### ISSUE TITLE
Get better results 2018

### ISSUE THREAD

**Issue by turner11** [turner11/VQA-MED#22]
| Loss function | activation | trainable parameters | BLEU | WBSS| DB ID |
| ------------- | ------------- | ------------- | ------------- | ------------- |------------- |
mean_absolute_percentage_error | relu | 147389 | 0.1594 | 0.3628 | 126 |"
mean_absolute_percentage_error | relu | 83901 | 0.1594 | 0.3628 | 121 |"
mean_absolute_error | relu | 147389 | 0.1525 | 0.3511 | 70 |"
mean_absolute_percentage_error | relu | 83901 | 0.1526 | 0.3494 | 125 |"
cosine_proximity | tanh | 83901 | 0.2132 | 0.1698 | 85 |"
cosine_proximity | sigmoid | 147389 | 0.2210 | 0.1693 | 77 |"
cosine_proximity | relu | 147389 | 0.2212 | 0.1679 | 98 |"
poisson | softmax | 147389 | 0.2151 | 0.1617 | 107 |"
